public-html/404.php

```diff
@@ -0,0 +1,13 @@
+<?php
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
+
+use template\Page;
+
+$new = new Page;
+$content = <<<EOT
+<div class="main_content">
+<h1 style="text-align:center; color:red"> 404 Page Not Found! </h1>
+</div>
+EOT;
+$new->constructPage($content);
```

Comment:
i guess time is right to check out response headers as well, Also try utilizing 404 as response header.

Reply:
you mean something like this ->
```php

<?php
header("HTTP/1.0 404 Not Found");
?>

```

Reply:
yup something like this check a bit more around this.

Reply:
```php
//Send 404 response to client.
http_response_code(404)
include 'error/404.php';
exit;
```

Reply:
What is the difference between ->
` header("HTTP/1.0 404 Not Found"); `
and 
` http_response_code(404) `